.emacs.d/lisp/code/smartparens-c.el

```diff
@@ -2,4 +2,13 @@
 
 (smartparens-global-mode t)
 
+(defun kotct/open-on-empty-line (&rest _ignored)
+  "Make sure the opening brace or bracket is placed on a new indented line.
+
+Intended for use as a pre-handler with smartparens."
+  (message "hi!")
```

Comment:
Still needed?

Reply:
fuck i knew i was going to forget this

Reply:
Actually I just ended up ditching this whole function since it wasn't working as I wanted. I'm just going to delete it.